### ISSUE TITLE
[ASK] Can envoy send grpc error code in REST response

### ISSUE THREAD

**Issue by syukur91** [envoyproxy/envoy#1908]
I have a case that when I send grpc request via REST the log said that it 200 code which mean success, but I'm not sure it's clearly success becaus when I look the grpc log it said error 401. So can envoy  send also grpc error code response?

**Comment by mattklein123** [comment]
I'm sorry I don't understand the question. Can you rephrase with more detail?

**Comment by mattklein123** [comment]
Closing this out. Feel free to reopen with more info.
